dev-python/kaptan/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>m.j.everitt@iee.org</email>
+		<name>Michael Everitt</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+</pkgmetadata>
```

Comment:
longdescription is missing, you can add the github upstream here too.

Reply:
Does this look better:
https://bpaste.net/show/2cc3dd0d0b68

Reply:
better, but I am not sure about the word "syntaxes"

Reply:
It doesn't sound right, I agree .. but it checks out .. https://www.thefreedictionary.com/syntaxes , https://simple.wiktionary.org/wiki/syntax , http://www.memidex.com/syntaxes etc

Reply:
Amended further, how's this:
https://bpaste.net/show/8cac80fca726